internal/health_checks/health_checks_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package health_checks_test
```

Comment:
```suggestion
package healthchecks_test
```

Reply:
Addressed in https://github.com/GoogleCloudPlatform/ops-agent/pull/1051/commits/781ce8b21adbc3763140d8da6f487f76a8aacd8d#diff-4d3b77bf3c7f6e9fcb06552c5495096ca5867f18d0647ec8b00c71523f8537f2 .